Stanford/JumpstartEngineering/Install/CAPx/CAPxDisplay.php

```diff
@@ -0,0 +1,116 @@
+<?php
+/**
+ * @file
+ * Define the context for the CAP display view mode for person pages.
+ */
+
+namespace Stanford\JumpstartEngineering\Install\CAPx;
+/**
+ *
+ */
+class CAPxDisplay extends \AbstractInstallTask {
```

Comment:
Hm. How far do we go with dependencies? This context assumes a lot of modules: entity, bean, field_collection, context, stanford_bean_types, on and on. Do we list everything?

Part of me thinks the only requirement for this particular class should be `context`.